avm/res/purview/account/main.bicep

```diff
@@ -189,29 +212,40 @@ module portal_privateEndpoints 'br/public:avm/res/network/private-endpoint:0.4.0
     privateDnsZoneResourceIds: privateEndpoint.?privateDnsZoneResourceIds
     roleAssignments: privateEndpoint.?roleAssignments
     tags: privateEndpoint.?tags ?? tags
-    manualPrivateLinkServiceConnections: privateEndpoint.?manualPrivateLinkServiceConnections
     customDnsConfigs: privateEndpoint.?customDnsConfigs
     ipConfigurations: privateEndpoint.?ipConfigurations
     applicationSecurityGroupResourceIds: privateEndpoint.?applicationSecurityGroupResourceIds
     customNetworkInterfaceName: privateEndpoint.?customNetworkInterfaceName
   }
 }]
 
-module blob_privateEndpoints 'br/public:avm/res/network/private-endpoint:0.4.0' = [for (privateEndpoint, index) in (storageBlobPrivateEndpoints ?? []): {
+module blob_privateEndpoints 'br/public:avm/res/network/private-endpoint:0.4.1' = [for (privateEndpoint, index) in (storageBlobPrivateEndpoints ?? []): {
   name: '${uniqueString(deployment().name, location)}-Storage-Blob-PrivateEndpoint-${index}'
   params: {
-    privateLinkServiceConnections: [
+    name: privateEndpoint.?name ?? 'pep-${last(split(account.id, '/'))}-${privateEndpoint.service}-${index}'
+    privateLinkServiceConnections: privateEndpoint.?manualPrivateLinkServiceConnections != true ? [
       {
-        name: name
+        name: privateEndpoint.?privateLinkServiceConnectionName ?? '${last(split(account.id, '/'))}-${privateEndpoint.service}-${index}'
         properties: {
           privateLinkServiceId: account.properties.managedResources.storageAccount
           groupIds: [
-            privateEndpoint.?service ?? 'blob'
```

Comment:
Hey @krbar was there a reason you removed the default's all over? Can this maybe only be `blob` for purview?

Reply:
@AlexanderSehr Updated to Private Endpoints variant 1, with the defaults

Reply:
I see where you're comming from. Just leaning towards variant one because the way the resources are named heavily implies they're created for a very specific service (e.g. `blob`)

Reply:
Thanks for recovering :)